pkg/gui/views.go

```diff
@@ -186,5 +188,28 @@ func (gui *Gui) createAllViews() error {
 	gui.Views.Snake.Title = gui.c.Tr.SnakeTitle
 	gui.Views.Snake.FgColor = gocui.ColorGreen
 
+	if gui.c.UserConfig.Gui.ShowPanelJumps {
+		jumpBindings := gui.c.UserConfig.Keybinding.Universal.JumpToBlock
+		jumpLabels := make([]string, len(jumpBindings))
+		for i, binding := range jumpBindings {
+			jumpLabels[i] = fmt.Sprintf("[%s] ", binding)
```

Comment:
Sounds reasonable. I had added the space here to avoid the +1/-1 changes in other places (you already noticed that I'm bad with off-by-one errors) but you're right that it doesn't result in a nice API.

And thanks for the `lo.Map` tip!